repos/spack_repo/builtin/packages/hip/package.py

```diff
@@ -462,16 +450,16 @@ def patch(self):
             )
         perl = self.spec["perl"].command
 
-        if self.spec.satisfies("@5.6:"):
+        if self.spec.satisfies("@5.7:"):
             with working_dir("clr/hipamd/bin"):
                 filter_file("^#!/usr/bin/perl", f"#!{perl}", "roc-obj-extract", "roc-obj-ls")
-        if self.spec.satisfies("@5.6:5.7"):
+        if self.spec.satisfies("@5.7"):
             with working_dir("hipcc/bin"):
                 filter_shebang("hipconfig")
 
         if self.spec.satisfies("+rocm"):
             numactl = self.spec["numactl"].prefix.lib
-            if self.spec.satisfies("@5.6:5.7"):
+            if self.spec.satisfies("@5.7"):
```

Comment:
Can we group these conditionals, and have a single line for the condition on `@5.7`?